docs/content.md

```diff
@@ -442,3 +469,99 @@ When updating not just the text content within the file, but the actual list of
 - Export the XLSX to CSV with filename `internship_providers.csv`
 - Place it in the `lib/tasks/support` directory.
 - Run `bundle exec rake teaching_internship_providers:generate`.
+
+
+## Creating a new page
+
+1. **Make sure you start by creating a new branch on master**
+To do this you will need to
+	- always open Visual Studio Code from https://github.com/DFE-Digital/get-into-teaching-app and open from the green code button
+	- once Visual Studio Code is open, click the search bar and select 'Create new branch'
+	- type in the name of your new branch and press enter
+	- if your branch has been created, it will appear as a name in the bottom left hand corner, next to the blue tab of the codespace name 
+
+3. Navigate to the right hand side bar, where you want the new page to sit
+4. Right hand click and select 'New file'
+5. Give name to file ie. when-to-apply.md **make sure you put md on the end of the name if it is a markdown file**
```

Comment:
how do you know if it's a markdown file? Is that just one with text as opposed to it being a component?

Reply:
Let's add this to the checklist for the next iteration!